src/main/java/frc/robot/commands/MoveToTarget.java

```diff
@@ -0,0 +1,80 @@
+package frc.robot.commands;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.path.GoalEndState;
+import com.pathplanner.lib.path.PathPlannerPath;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.subsystems.SwerveSubsystem;
+import frc.robot.util.MiscUtils;
+import java.util.List;
+
+public class MoveToTarget extends Command {
+  private Pose2d absolutePose;
+  private SwerveSubsystem swerve;
+
+  PathPlannerPath constructedPath;
+  Command pathCommand;
+
+  private MoveToTarget(SwerveSubsystem swerve, Pose2d absolutePose, boolean reflected) {
+    this.swerve = SwerveSubsystem.getInstance();
+    if (reflected) {
+      this.absolutePose = MiscUtils.reflectAcrossMidline(absolutePose);
+    } else {
+      this.absolutePose = absolutePose;
+    }
+    addRequirements(swerve);
+  }
+
+  @Override
+  public void initialize() {
+    // constructing the list of path points using absolute coordinates on the field
+    Pose2d currentPose = swerve.getState().Pose;
+    List<Translation2d> bezierPoints = PathPlannerPath.bezierFromPoses(currentPose, absolutePose);
+
+    // create the path using the path points and constraints (also providing the final robot
+    // heading)
+    constructedPath =
+        new PathPlannerPath(
+            bezierPoints,
+            Constants.PPConstants.PATH_PLANNER_CONSTRAINTS,
+            new GoalEndState(0.0, absolutePose.getRotation()) // goal end velocity and heading
+            );
+
+    // prevent automatic path flipping by AutoBuilder (we want to execute absolute path)
+    constructedPath.preventFlipping = true;
+
+    // Command of the built auto path
+    pathCommand = AutoBuilder.followPath(constructedPath);
+    pathCommand.initialize();
+  }
+
+  public void execute() {
+    if (pathCommand != null) {
```

Comment:
consistency with the `end()` command; either have the curly braces for both or neither

Reply:
alright done